src/ws/cockpitauth.c

```diff
@@ -1147,6 +1158,7 @@ cockpit_session_launch (CockpitAuth *self,
       if (!host)
         host = type_option (COCKPIT_CONF_SSH_SECTION, "host", "127.0.0.1");
 
+      capture_stderr = cockpit_conf_bool (COCKPIT_CONF_SSH_SECTION, "reportstderr", FALSE);
```

Comment:
TBH I'm not happy for this to be a config option, at least at first sight. Is there any harm in just always capturing stderr? That seems useful for "regular" bastion host/remote host usage as well. I suppose with cockpit-ssh we have tight control about stderr and there's usually nothing interesting on it (as it speaks the json protocol), but then it's at most "harmless".

Then the login page can decide for itself in which mode it is, and show stderr if it is in client mode. I realize that we might not want to do that unconditionally for web mode, in order to not give away too much information on login failures -- but as the login page is already different in client mode, it already knows whether or not its appropriate to show ssh stderr?

Reply:
As I said in https://github.com/cockpit-project/cockpit/pull/16629#discussion_r754859348, I'd be happy to temporarily use an undocumented [WebService] _Internal_Client = yes flag, and merge this into RequireHost= later. But at first sight, reporting ssh stderr with `RequireHost=!no` (i.e. "we are doing something with ssh") might not be too wrong?

But either way, to me this doesn't feel right for a config option, as it's not really a static decision. auth.c should know which auth mode it is doing, and whether it's appropriate to expose stderr. 

As this is a bit hairy (give enough information vs. avoid information leaks), maybe putting this into `_Internal_Client` for now is not the worst start?